### DISCUSSION TITLE
Alternative to Elasticsearch

### DISCUSSION THREAD

**Discussion by gururajshivananda** [open-metadata/OpenMetadata#9047] [General]
Elasticsearch has recently changed its license for  commercial Apps bundling it. Does Open metadata have option to choose a different open source search engine like opensearch

**Comment by harshach** [comment]
@gururajshivananda we currently work with OpenSearch as well.

**Reply by gururajshivananda** [reply]
can u point to documentation for using opensearch with Openmeatadata
